websockify/websocketproxy.py

```diff
@@ -456,20 +456,20 @@ def websockify_init():
             "directory containing configuration files of this form "
             "(DEPRECATED: use `--token-plugin TokenFile --token-source "
             " path/to/token/file` instead)")
-    parser.add_option("--token-plugin", default=None, metavar="PLUGIN",
-                      help="use the given Python class to process tokens "
-                           "into host:port pairs")
+    parser.add_option("--token-plugin", default=None, metavar="CLASS",
+                      help="use a Python class, usually one from token_plugins.py, "
```

Comment:
The syntax used is the Python import syntax, so we should probably use that here as well. I.e. `websockify.token_plugins` rather than `token_plugins.py`.

Reply:
Referring to the files by name makes it a bit easier for users to figure out which options are available for --token-plugin and --auth-plugin, especially if they don't know python.

Reply:
Possibly. I guess it highlights a need for more detailed documentation than just --help output..

Reply:
Yeah, I might write a wiki page about plugins.

Reply:
We have this:

https://github.com/novnc/websockify/wiki/Token-based-target-selection

But it might be better to have documentation in the source tree. People might not have access to the wiki when they are configuring things. (or know about it)

Reply:
A brief paragraph in the readme explaining where to find the plugins, then?

Reply:
Check the latest commit, I think it's fine now.